Orangenspannung/oestersjoen
# O02 create russian states

# What has changed:
* Created the Russian states and gave them their territory
# How to test:
* Open the game and look at Russia, you should see 10 new countries
* Check if the soviet union still hides anywhere, they should be gone.
* Check if you find any major bugs.
